webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js

```diff
@@ -29,6 +29,8 @@ export class SakaiGrader extends gradableDataMixin(SakaiElement) {
     this.hasUngraded = false;
     this.hasUnsubmitted = false;
     this.resubmitNumber = "1";
+    this.previousPrivateNotes = "";
```

Comment:
You shouldn't need to do this. We already store the original submissions and the Submission class has a privateNotes field. We need to try avoid duplicating data structures.

Reply:
I shaw that, you are right! That was not necesary! Thanks!